docs/PT-BR/EXAMPLES.md

```diff
@@ -66,11 +66,33 @@ describe("foo", () => {
 });
 ```
 
+## waitForAlertToBePresent
+
+| 1 argumento opcional                   |
+|----------------------------------------|
+| `timeoutInMilliseconds` (padrão: 5000) |
+
+```js
+const protractorHelper = require("protractor-helper");
+
+describe("foo", () => {
+  it("bar", () => {
+    browser.get("https://example.com");
+
+    const alertElement = element(by.className("el-alert"));
```

Comment:
This needs to be adjusted as in the English version.

Reply:
done